src/DateSelector/index.js

```diff
@@ -370,6 +372,10 @@ DateSelector.propTypes = {
    * Triggered when a preset is selected.
    */
   onPresetChange: func,
+  /**
+   * @see [Popover](#popover) - The placement to be used on Popover component
```

Comment:
This description should be more specific.
```suggestion
   * @see [Popover](#popover) - The placement to be used on Popover component, this will change the position of the calendars to open from de left of the base input(default position) or from the right side of the base input. 
```

Reply:
Plz review the text, because of typo 😄

Reply:
Description improved! I changed yours a little bit on the latest push 🙂